content/blog/2023/2023-01-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Jenkins December 2022 Newsletter"
+tags:
+- jenkins
+- newsletter
+- community
+- contribute
+authors:
+- alyssat
+opengraph:
+image: /images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png
+links:
+discourse: true
+---
+
+== December 2022
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/Recap.png[Jenkins 2022 recap Newsletter]
+
+Welcome to the Jenkins Newsletter!
+This is a compilation of progress within the project, highlighted by Jenkins Special Interest Groups (SIGs) for December 2022.
+
+Happy reading!
+
+Got Inspiration?
+We would love to highlight your cool Jenkins innovations.
+Share https://docs.google.com/forms/d/e/1FAIpQLScMCGOMtn2hGpfXsbyssGhVW1LwlW4LkXCIaKINKDQU2m6ieg/viewform[your story] and you could be in the next Jenkins newsletter.
+
+== Highlights
+
+* Welcome Dr. Ullrich Hafner, Alexander Brandes and Kevin Martens as new members to the governance board and documentation officer.
+* Platform modernization
+** Require Java 11. https://www.jenkins.io/blog/2022/12/14/require-java-11/[Build toolchain changes] arrived in parent pom 4.52
+** Fully supports Java 17
+** Table layouts, menu entries, icons, themes, breadcrumbs, and more have been updated to give Jenkins a new, fresh look in 2022.
+* The Pipeline Steps reference guide has been improved.
```

Comment:
```suggestion
** Table layouts, menu entries, icons, themes, breadcrumbs, and more have been updated to give Jenkins a new, fresh look in 2022.

* The Pipeline Steps reference guide has been improved.
```
just trying to get this list to display properly, in the preview there is a line break between the table layouts item and the pipeline steps item